LSF/bjobGen.sh

```diff
@@ -0,0 +1,226 @@
+#!/bin/bash
```

Comment:
This file doesn't belong in a public repository, it's not going to work for others as it's expecting thing sin home directories and module to be available.  You are setting yourself up for github issues as to how to use this.